tests/x11/remote_desktop/windows_client_remotelogin.pm

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
```

Comment:
Is it old license? i guess use the license come from
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6350/files#diff-60f9c251c12e16c3612e50969c83ae89

Reply:
Updated